sksurv/meta/ensemble_selection.py

```diff
@@ -63,6 +63,9 @@ def fit(self, X, y=None, **kwargs):  # pragma: no cover; # pylint: disable=unuse
     def predict(self, X):  # pylint: disable=no-self-use
         return X.mean(axis=X.ndim - 1)
 
+    def predict_survival_function(self, X):
+        return X.mean(axis=X.ndim - 1)
```

Comment:
Taking the average over survival functions is problematic, because you could end up with an average survival function that is not monotonically decreasing.